specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-08-01-preview/machineLearningServices.json

```diff
@@ -860,6 +860,116 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/computes/{computeName}/getAllowedVmSizesForResize": {
+      "post": {
+        "tags": [
+          "OperationalizationClusters",
+          "MachineLearningComputes"
+        ],
+        "operationId": "Compute_getAllowedResizeSizes",
+        "description": "Returns supported virtual machine sizes for resize",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ComputeNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/VirtualMachineSizeListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List VM Sizes": {
+            "$ref": "./examples/Compute/getAllowedVMSizesForResize.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/computes/{computeName}/resize": {
+      "post": {
+        "tags": [
+          "OperationalizationClusters",
+          "MachineLearningComputes"
+        ],
+        "operationId": "Compute_Resize",
+        "description": "Updates the size of a Compute Instance.",
+        "x-ms-long-running-operation": true,
```

Comment:
>"x-ms-long-running-operation": true [](http://example.com/codeflow?start=8&length=35)

A LRO Post operation with return schema must have "x-ms-long-running-operation-options" extension enabled.

[https://github.com/Azure/azure-openapi-validator/blob/main/docs/long-running-operations-options-validator.md](https://github.com/Azure/azure-openapi-validator/blob/main/docs/long-running-operations-options-validator.md)

Reply:
Well, I don't want to have a return schema. I was forced to add the 200 response because there is a LintDiff rule that says that LRO are required to have a 200 and 202. Ideally I want to not have the 200 return at all. Can I have that and you bypass the rule?

Reply:
If the LRO-Post does not have a schema, please remove 200 and only define 202. If there is still a lint-diff error, please follow https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/85/Swagger-Suppression-Process to suppress it.

Reply:
The existing LROs do not have 200 and 202, but there is also not a single suppression defined in the Machine Learning Repo that I could find. I find it hard that this would be the first time that someone needs to add an exception in MachineLearningServices. Am I missing something?

Reply:
The existing content will not be validated by the lint-diff. Only newly added content will be validated.

Reply:
Removed 200, added suppression